pkg/build/api/types.go

```diff
@@ -65,19 +75,69 @@ type BuildSpec struct {
 	// Output describes the Docker image the Strategy should produce.
 	Output BuildOutput
 
-	// Compute resource requirements to execute the build
+	// Compute resource requirements to execute the build.
```

Comment:
To make this valid godoc it should start with "Resources controls the compute resources...."